### DISCUSSION TITLE
Dịch "pointwise mutual information"

### DISCUSSION THREAD

**Discussion by rootonchair** [d2l-ai/d2l-vi#3025] [Q&A] [answered]
Hiện có 2 cách dịch:
* anh @quangnhat185 : `thông tin tương hỗ cấp điểm`. Em nghĩ nó bắt nguồn từ việc em dịch `point level` ra thành `cấp điểm`
* anh @LongNguyen1984 : `thông tin tương hỗ điểm-điểm`

@duythanhvn @cuongvng @lkhphuc @davidnvq @minhduc0711

**Comment by lkhphuc** [comment]
Các từ `element-wise`, `position-wise` hiện mình đang dịch là `theo từng phần tử / theo vị trí`.
Vậy từ này cứ giữ kiểu: thông tin tương hỗ theo từng điểm`?

**Comment by quangnhat185** [comment]
Mình thì thấy từ `cấp điểm` đọc thuận miệng hơn.  Các cách dịch khác vẫn bảo toàn được nghĩa chính nên mình ko có preference gì. Thôi thì mn vote rồi mình thống nhất một cách dịch :

:+1:  `thông tin tương hỗ cấp điểm`
:heart: `thông tin tương hỗ điểm-điểm`
:tada: `thông tin tương hỗ theo từng điểm`

@rootonchair @LongNguyen1984  @lkhphuc  @minhduc0711  @trGiang99 @davidnvq @cuongvng @duythanhvn

**Reply by minhduc0711** [reply]
mình nghĩ cứ flexible cũng được, `pointwise` thì dịch là `theo từng điểm` còn `point-level` thì dịch là `cấp điểm`

**Comment by thanhcsf** [comment, marked as answer]
Vậy chỗ này chốt sử dụng **thông tin tương hỗ theo từng điểm** anh em nhé.
